src/transducers.js

```diff
@@ -214,3 +214,25 @@ class Tap extends Transducer {
     return this.xfStep(value, input)
   }
 }
+
+export const interpose = separator => xf => new Interpose(separator, xf)
+class Interpose extends Transducer {
+  constructor(separator, xf) {
+    super(xf)
+    this.separator = separator
+    this.started   = false
+  }
+  [tStep](result, value) {
```

Comment:
I've been using `(value, input)` here.  The `input` taken from the "input source" as described in Clojure's documentation, and wanted something different than `result` to distinguish it from the `result` function.  Likewise `[tResult](value)`.  I'm open to changing it, but would like to be consistent.